
#### hormone receptor

There are genomic basis of ER+ breast cancer types. Since some tumors progress regardless if ER+ inhibitors are applied.

ER+ metastatic breast cancer is dominated by mutations in PI3K pathway. Genomic alterations in EGF signaling promote endocrine resistance. HER2- subgroup has worse prognosis. estrogen receptor is a dominant alteration from primary and metastatic samples. ESL1 mutations are diverse and not entirely clonal.

#### activation and nuclear crosstalk   
NR3C and ER family receptors diverged quite some time ago. NR3C includes progestrogen receptor, androgen receptor, glucocorticoid receptor. NR3C activation results in altered ER chromatin association and gene expression. Conversely, ER activation alters NR3C chromatin accessibility. Common themes of ER3C receptors
+ may form complex at ER
+ exhibit breast cancer subtype-dependent effects on prognosis
+ activity also dependent on stage

_context NR3C signaling depends on tumor ER status_
+ Better prognosis with ER+ NR3C+ than ER+ NR3C-.
+ NR3C biology depends on stage

#### Non coding sequences
Profiling mutations informs the tumor biology and can serve to monitor response to treatment.

One study is temporal tracing of alterations in blood samples using liquid biopsy and DNA sequencing. Mutations are already established 11 months before recurrences. Inter-patient heterogeneity raises the need for comprehensive set of mutations to profile. Noncoding somatic mutations offer an opportunity to expand the list of driver mutations because non coding sequence constitute the majority of mutations. Non coding sequence is a rich source of functional elements. Majority of these mutation occurs in heterchromatin, which is repressed and not important. But enhancer, promoter, insulator, repressing, repeats, and transcribed elements are important. Gene expression relies on chromatin interactions, by constructing spatial proximity to regulate a given gene of interest.

_So can driver noncoding mutations be found near oncogenes and suppressors?_
Hypersensitive site, promoters, and regulatory elements around ESR1 identified. tool offers a method to calculate statistical enrichment of mutations in SRE, or set of regulatory elements. To confirm enrichment, mutations in SRE of ESR1 are identified in an independent cohort of samples. mutations are predicted to alter the binding of transcription factors regulating ESR1 expression, most of the TF altered are well studied binding partners. Most mutations are gain of function mutations, as identified by luciferase assay. CRISPR/Cas9 deletion of mutated enhancers reduces ESR1 expression.

_Noncoding inherited risk-variants for breast cancer preferentially map to enhancers_
rs2046210 are putative causal SNPs of the risk-locus. Somatic mutations and genetic variants converge on one ESR1 regulatory element. it also disrupts binding of GATA3 to chromatin.

summary
1. gain of function noncoding mutation enrich within the set of regulatory elements (SRE) of the ESR1 gene in breast cancer.
2. somatic mutation and genetic variants converge on SRE
