


#### Cancer genomics: A translational future  Elaine Mardis

__Immunotherapy__  
+ NGS -> identify cancer unique peptides
+ HLA haplotypes of individual patients
+ RNAseq from cancer cells to identify genes that are mutated and expressed  

_Genome-guided cancer vaccine design: maleonma_

Genomic heterogeneity > widow down the number of possible differentially altered genes
Engineer neoantigens, CD8 T cells were upregulated, its clonal type expanded
Personallized Vaccine trial in TNBC
+ sequence exome
+ RNAseq
+ neoantigen peptides

Temporal change in cancer progression
+ neoantigen landscape evolve cross time.


#### Targeting epigenomic dependencies in cancer James Bradner

ex. myc tp53 are master regulators that cause cancer progress.

Small molecule tool box of writer, reader, eraser in the epigenome. Some of these participating enzymes are required for certain cancer. For example, myc does not act lone to activate proliferation pathways but instead brings to site a host of factors that leaves memory marks. Histone acetyltransferase modifies histone tails. These markers identified by bromodomains of BRDA. Bromodomain is a bundle of alpha helix, a helix can be bound inside its pocket. It would be enticing to make inhibtors of BRDA. Is it possible to erase markers of BRD4. Methylation is druggable and targetable. Myc was seen as the only downregulated oncogene after 2 hours of treatment. Computational modeling identified a master switch. Clinical translation of these drugs, bromodomain inhibitors, was very successful.

_Epigenomic resistance to epigenetic treatment_
TNBC: clones that grows faster in the presence of BRDA inhibitor. TNBC convert back to luminal type, upregulating genes necessary to inhibit effects of BRDA inhibitor. Engineer molecules that one arm binds BRD4 and the other arm binds E3 ligase, leading to its degradation. transcription collapses when BRD4 is degraded; epigenomic resistant breast cancer die.

#### Genome-editing using CRISPR-Cas systems Feng Zheng  

Genome editing using nucleases: targeted dsDNA induce repair pathway, during which insertion/deletion is introduced. Therefore loss-of-function mutation can be easily introduced.

_CRISPR:RNA-guided DNA recognition_
from the RNA-mediated bacterial adaptive immune system. Type II CRISPR cleaves exogenous DNA. (Garneau 2010) Program the guided RNA and utilizing catalytic enzymes for precise dsDNA breaks. Can also be used for DNA binding at promoter for degradation.

_studying genes at genome scale_
Targeting different places of the sequence. By using high throughput oligosynthesis, able to target every gene in the genome.
+ guide library synthesis
+ convert guide RNA plasmid library
+ introduce to bacteria

Loss-of-function or gain-of-function genetic screening

_Screen for drugs for melanoma drug resistant genes_
+ BRAF mutated
+ Vemurafenib: targets BRAF mutants.

Synergistic activation mediator: assemble SAM complex, inducing transcription upregulation.
+ computation design of guide sequences
+ identify targets

_Non coding sequences_
+ design of guide to target genome region immediately adjacent to gene.
+ synthesis sgRNA and produce lentivirus
+ transduce into BRAF mutant cells
+ treat with BRAF mutants
+ identify target motifs that may be known or novel

_discovering new enzymes for nature's discovery_
there are 2 class of CRISPR: class I and class II. Class utilizes multi-subunit crRNA-effector complex. Computational identification of novel class II like systems. Cpf1 is a potential enzyme replacing Cas9. Cpf1 is a simpler system and can be used for multiplex genome editing using a CRISPR tandem array. Cpf1 structure identifies a novel second nuclease domain, which has known homology with other nuclease domains. Future direction has been the identification of other CRISPR systems.

Identification of protein always present next to CRISPR may yield novel potential candidate systems. One of them C2c2 does not contain DNAase domain but instead contain 2 RNAase domain, may a an RNA targeting domain. This locus is expressed by RNAseq. Verified that C2c2 mediates defense against the RNA phage MS2. Identification of bacteria that survived phage infection and sequencing of those survived cells to identify C2c2 and CRISPR. Also, analyzing the parameters that are necessary for C2c2. C2c2 requires CAM for activation. C2c2 can be reprogrammed to knockdown RFP in E. coli. Therefore its transcriptome editing.

#### T cell recognition of human cancer Ton Schumacher from NCI
Value of antigenous tumor-specific T cell responses
+ TIL therapy
+ Checkpoint blockade

_nature of antigen of antigens that drive efective cnacer immuotherapy_

Major class of human cancer antigens
+ self antigens, to which tolerace is not complete (T Boon T Wolfei)  
+ neo-antigens, epitope that arise as a consequence of cancer-specific mutations (R Schreiber U Sahin)

_High throughput analysis of neo-antigen specific CD8 T cell responses_      
+ gnereate map of tumor specific mutations
+ determine which mutated genes are expressed (RNAseq)
+ predict epitopes for each mutation / each HLA-allele in silico
+ Screen for T cell recognition of mutated phenotype
+ infusion of neo-antigen specific T cells elicit elevated activity

_extent of DNA damage correlates with clinical benefits of cancer Immunotherapy_
Mutational load correlates with clinical outcome to anti-PD-1 in NSCLC


#### Stem cells and Cancer Elaine Fuchs

Cancer stem cels generate and fuel cancers. Niche-stem cell interactions decide stem cell activity.

Hair follicle stem cells go through synchronized, cyclical bouts of quiescence and activity: the hair cycle. 
