

---

UMBRELLA article   
[__The Molecular Pathology of Breast Cancer Progression__](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3069504/)  
The current model of human breast cancer progression proposes a linear multistep process which initiates as flat epithelial atypia (FEA), progresses to atypical ductal hyperplasia (ADH), evolves into DCIS, and culminates in the potentially lethal stage of invasive ductal carcinoma. For several decades a major challenge to human breast cancer research has been the identification of the molecular alterations associated with the different stages of breast cancer progression. Until recently, progress in attaining this goal has been hampered by technical limitations associated with applying advanced molecular technologies to the microscopic preinvasive stages of breast tumorigenesis. Recent advances in comprehensive, high-throughput genetic, transcriptomic and epigenetic technologies in combination with advanced microdissection and ex vivo isolation techniques have provided for a more complete understanding of the complex molecular genetic and molecular biological inter-relationship of the different stages of human breast cancer evolution. Here we review the molecular biological data that suggest breast cancer develops and evolves along two distinct molecular genetic pathways. We also briefly review gene expression and epigenetic data that support the view of the tumor microenvironment as an important coconspirator rather than a passive bystander during human breast tumorigenesis.


[__Comparative genomic hybridization of ductal carcinoma in situ of the breast-evidence of multiple genetic pathways__](http://www.ncbi.nlm.nih.gov/pubmed/10398097)


[__Gene expression profiles of human breast cancer progression__](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC156311/)    
Although distinct pathological stages of breast cancer have been described, the molecular differences among these stages are largely unknown. Here, through the combined use of laser capture microdissection and DNA microarrays, we have generated in situ gene expression profiles of the premalignant, preinvasive, and invasive stages of human breast cancer. Our data reveal extensive similarities at the transcriptome level among the distinct __stages__ of progression and suggest that gene expression alterations conferring the potential for invasive growth are already present in the preinvasive stages. In contrast to tumor stage, different __tumor grades__ are associated with distinct gene expression signatures. Furthermore, a subset of genes associated with high tumor grade is quantitatively correlated with the transition from preinvasive to invasive growth.

Our analysis also identified a subset of genes with quantitative expression levels that correlate with advanced tumor grade and with the transition from DCIS to IDC (Fig. ​(Fig.44A and Table 2). This observation is provocative in that it provides an apparent link between tumor stage and tumor grade.

![DCIS > IDC alteration](http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop.fcgi?p=PMC3&id=350573&s=10&r=1&c=4)

[__Gene expression profiling of the tumor microenvironment during breast cancer progression__](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687710/)

[__Molecular Markers in Ductal Carcinoma in Situ of the Breast__](http://mcr.aacrjournals.org/content/1/5/362.long)  
Gene expression patterns in ductal carcinoma in situ (DCIS), and in invasive, and metastatic breast tumors were determined using serial analysis of gene expression (SAGE). We used mRNA in situ hybridization to examine gene expression at the cellular level and immunohistochemistry on tissue microarrays to determine __association__ between gene expression patterns and histopathologic characteristics of the tumors. We found that that the most dramatic transcriptome change occurs at the __normal to DCIS transition__, while there is no clear universal “in situ” or “invasive” tumor molecular signature. From the 16,430 transcripts analyzed, we identified only __5 and 11__ that were preferentially up-regulated in DCIS and invasive tumors, respectively. The majority of invasive cancer specific SAGE tags correspond to novel genes. The genes we identified may define biologically and clinically meaningful subgroups of DCIS with a high risk of progression to invasive disease.

we found that the most dramatic difference in gene expression patterns occurs at the normal to in situ carcinoma transition and it involves the __uniform down-regulation__ of 34 genes.  We also identified 144 genes __up-regulated__ in a fraction of in situ and invasive or metastatic tumors. There is not a pronounced and common “in situ,” “invasive,” or “metastasis” signature. Correlating with this observation using clustering and other statistical analyses, we were __not able to__ identify any gene that was universally and specifically up- or down-regulated in DCIS or invasive tumors
