

[__CRSPR__](https://en.wikipedia.org/wiki/CRISPR)  
Clustered regularly-interspaced short palindromic repeats are segments of prokaryotic DNA containing short repetitions of base sequences. Each repetition is followed by short segments of "spacer DNA" from previous exposures to a bacterial virus or plasmid. The CRISPR/Cas system is a prokaryotic immune system that confers resistance to foreign genetic elements such as plasmids and phages,and provides a form of acquired immunity .CRISPR spacers recognize and cut these exogenous genetic elements in a manner analogous to RNA interference in eukaryotic organisms.

![crspr immune system](https://upload.wikimedia.org/wikipedia/commons/5/5f/Crispr.png)





[__HISTORY__](https://www.broadinstitute.org/what-broad/areas-focus/project-spotlight/crispr-timeline)  

_1987_ [__First description of CRSPR in E. coli__](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC213968/))  
Sequencing of IAP gene and confirmation of its protein product. An unusual structure found at 3-end flanking region of IAP was noted. Five highly homologous sequences of 29 nt were arranged as direct repeats with 32 nucleotides long. The conserved sequence is palindromic([_dyad symmetry_](https://en.wikipedia.org/wiki/Dyad_symmetry)) and was named as __REP__ sequence, which was found in both E. coli and Salmonella. It is hypothesized to stabilize mRNA.

---
_2002_ [__Identifiction of CAS__](http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.2002.02839.x/full)  

_ABSTRACTS_

+ In most species with two or more CRISPR loci, these loci were flanked on one side by a common leader sequence of 300–500 b. The direct repeats and the leader sequences were conserved within a species, but dissimilar between species.  
+ Four CRISPR-associated (cas) genes were identified in CRISPR-containing prokaryotes that were absent from CRISPR-negative prokaryotes.  
+  The cas genes were invariably located adjacent to a CRISPR locus, indicating that the cas genes and CRISPR loci have a functional relationship. The cas3 gene showed motifs characteristic for helicases of the superfamily 2, and the cas4 gene showed motifs of the RecB family of exonucleases, suggesting that these genes are involved in DNA metabolism or gene expression.  

_RESULTS_  

_Disclosure of CRISPR motifs in sequence databases_  

Identification of CRISPR motifs using query against GenBank  

_Comparison of the repeat and spacer sequences in the various prokaryotes_

The repeat sequences in the CRISPR loci of closely related species were similar or identical. Despite the absence of sequence similarity between the repeat sequences of the species, the repeat se-quences share some common features. In most of the repeat sequences a lose dyad symmetry can be recognized. In most organisms, the last repeat or the few last repeats of a CRISPR locus contained mutations.The DNA stretches between the repeats (the spacers) were present as a single copy in a CRISPR locus with very few exceptions.

_Identification of common sequences flanking CRISPR loci_   

These leader sequences were several hundred base pairs long. They were located on one side of the CRISPR loci and their orientation with regard to the repeat sequence was invariably the same. The nucleotide sequences of the leaders within a given species shared approximately 80% sequence identity, however no homology was found among the leaders of taxonomically unrelated species.

_Identification of CRISPR associated genes_  

Comparison of the genes that flank the CRISPR loci in the genomes of different prokaryotic species showed a clear homology among four genes. We designated these genes the CRISPR-associated genes, cas1 to cas4. The Cas proteins whosed some amazing homology. The amino acid sequences of the Cas proteins show some highly conserved amino acid residues or functional domains.

![CAS table](http://api.onlinelibrary.wiley.com/asset/v1/doi/10.1046%2Fj.1365-2958.2002.02839.x/asset/image_n%2FMMI_2839_f1.gif?l=q8hDySOaNtvIQToMM6q%2Bhx2wvVXtg3EYc7tKU92jLUGkSReL9hjUH19Go3%2BfxZawZOQgSA3HEnY7%0AsaSUY4iCRA%3D%3D&s=%2294c5b32a1dab4586b545b990f337fee9%22&a=wol)  

_Comparison of the cas genes and Cas proteins_

The cas genes were found to be located on either side of the CRISPR locus, and no preference for direction of the cas reading frames was observed. The cas genes of most species are present in the genome in the order cas3–cas4–cas1–cas2. The cas gene cluster generally was found to be located within a few hundred of basepairs of the CRISPR locus.

+ The predicted __Cas3__ proteins showed the seven functional domains of the superfamily 2 of helicases, indicating that the Cas3 homologues are members of this group of DNA-modifying proteins
+ The __Cas4__ proteins showed similarity to the family of RecB exonucleases.


_CONCLUSIONS_

In this study we identified four CRISPR-associated genes, cas1 to cas4. The presence of the cas genes was invariably associated with the presence of CRISPR loci in the genome. The cas1 gene was found in all species with CRISPR loci, whereas the other three cas genes were present in most but not all CRISPR-containing genomes. The strict association between CRISPRs and cas homologues in sequenced genomes is highly suggestive for a functional relationship between these genes and the CRISPRs.

---

_2005_ [__Ubiquity of CRSPR and homology in bacteriophage sequences__](http://link.springer.com.myaccess.library.utoronto.ca/article/10.1007/s00239-004-0046-3/fulltext.html)  

_ABSTRACT_  

Prokaryotes contain short DNA repeats known as CRISPR, recognizable by the regular spacing existing between the recurring units. Discovery of its __origin__ may provide clues on its biological function. Showed that __CRISPR spacers__ are derived from preexisting sequences, either chromosomal or within transmissible genetic elements such as bacteriophages and conjugative plasmids. Remarkably, these extrachromosomal elements fail to infect the specific spacer-carrier strain, implying a relationship between CRISPR and immunity against targeted DNA.

_INTRO_

+ They consist of 24- to 40-nucleotide (nt) recurrent motifs regularly spaced by intervening sequences of sizes similar to that of the repeated unit and named CRISPR (clustered regularly interspaced short palindromic repeats)  
+ Comparative study of theses peculiar elements in different prokaryotic lineages, using computer query on existing genome sequences: They represent the most widely distributed family of repeats among prokaryotic genomes.   
+ Biological function is predicted by the broad distribution and the remarkable structural conservation of CRISPR, but this has not been firmly established.
+ Interestingly, these targeted viruses are unable to infect the spacer-carrier cell but succeed with closely related strains lacking the specific CRISPR spacer.

_METHODS_

__PCR ampolification__ of E. coli CRISPR loci, and then sequencing  
__Sequence Analysis__ of CRISPR loci of available prokaryotic genomes

_RESULTS_

+ identified about 4500 CRISPR spacer from 67 strains
+ spacer homolog identified in phages

---

_2005_ [__Discovery of Cas9 and PAM__](http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.28048-0#tab2)   

_ABSTRACT_    

+ In silico analysis revealed another cluster of three genes associated with CRISPR structures in many bacterial species, named here as cas1B, cas5 and cas6, and also revealed a certain number of spacers that have homology with extant genes, most frequently derived from phages, but also derived from other extrachromosomal elements   
+ Phage sensitivity of S. thermophilus strains appears to be correlated with the number of spacers in the CRISPR locus the strain carries.   
+ The authors suggest that the spacer elements are the traces of past invasions by extrachromosomal elements, and hypothesize that they provide the cell immunity against phage infection, and more generally foreign DNA expression, by coding an anti-sense RNA.  
+ The presence of gene fragments in CRISPR structures and the nuclease motifs in cas genes of both cluster types suggests that CRISPR formation involves a DNA degradation step.  

_INTRO_

+ We report here that many spacers have homology with known genes, most often derived from extrachromosomal elements such as phages and plasmids. We propose that the formation of CRISPR structures involves DNA fragmentation by cas genes, and that the apparent stability and widespread presence of CRISPRs in bacterial genomes may be due their protective function against foreign DNA invasion.

_RESULTS_  

_The S. thermophilus CRISPR locus and associated genes_

42 repeats of 36 bp, GTTTTTGTACTCTCAAGATTTAAGTAACTGTACAAC, separated by unique sequence spacers of 30 bp. A gene localized upstream of the locus, str0658, encodes a homologue of the Cas1 protein

![S thermophilus CRISPR](http://mic.microbiologyresearch.org/docserver/fulltext/micro/151/8/MIC65355-1.gif)

`tr0658` and the two flanking genes, `str0657` and `str0659`, which we term herewith cas5 and cas6, respectively, have homologues that are clustered in numerous bacterial species, belonging to widely divergent phylogenetic groups. Furthermore, the cas1 genes group in two __clusters__, cas1A and cas1B (Fig. 2b F2 ); the former is always associated with cas2, cas3 and cas4, and the latter with cas5 and cas6 genes.   

![cas1 cross lineages](http://mic.microbiologyresearch.org/docserver/fulltext/micro/151/8/MIC65355-2.gif)  

cas1B, cas5 and cas6 genes are localized downstream of the CRISPR cluster. The __Cas5 family groups__(Cas9) large proteins (>1100 aa) that carry an __HNH motif__ present in various nucleases, including colicin E9, which causes cell death by introducing double-stranded breaks into DNA, and a number of restriction enzymes  

_CRISPR spacers have homology with extant genes_   

CRISPR loci were found close to cas1 genes in about 50 of the 198 complete genomes available in the NCBI database. They totalled 2156 spacers, 44 of which are homologous to other genes. The 44 spacers are carried in 4 archaeal and 10 bacterial species, spanning a broad phylogenetic range. Remarkably, six of these share homology with genes that reside in the vicinity of extrachromosomally derived genes, three share homology with genes of __aberrant G+C__ content and two share homology with genes from regions where the gene order differs from that of phylogenetically close genomes

> Horizontal transfer could lead to the gene organization observed in all these cases.

_CRISPR alleles in different S. thermophilus strains_  


analysed the CRISPR alleles of 22 S. thermophilus and 2 S. vestibularis strains considering phages of lactic acid bacteria are among the best characterized in respect of genome data and that lactic acid bacteria are exposed frequently to phage attacks under the conditions of dairy fermentations. The number of repeats varied between 10 and 51 for different CRISPR alleles. The repeats were strictly identical, with the exception of 38 out of a total of 632 (6 %). The length of spacers comprised between 28 and 32 bp, being 30 for 556 of the 618 spacers (90 %). The total length of the CRISPR loci (from the first nucleotide of the first repeat to the last nucleotide of the last repeat) varied between 628 and 3404 bp.

_S. thermophilus spacers are homologous to extrachromosomal elements_


Out of 349 unique spacers some 124 (36 %) had significant matches (E<0·001) with sequences in the NCBI nucleotide database. The best matches were with phages of streptococci (75 %), and plasmids of S. thermophilus or Lactococcus lactis (20 %).  


_Phage resistance of S. thermophilus is correlated with the number of spacers in a CRISPR locus_  

As many spacers in the CRISPR loci have homology with phage sequences, we searched for a correlation between CRISPR properties and a phage-related phenotype. A negative correlation between the number of spacers at a locus and the sensitivity of the strain to phage infection (expressed as a proportion of phages able to propagate on a strain) is observed

![number of spacer correlates with phage resistance](http://mic.microbiologyresearch.org/docserver/fulltext/micro/151/8/MIC65355-6.gif)  

_DISCUSSION_

+ Horizontal gene transfer may underlie all these cases, suggesting that incorporation of gene fragments into CRISPR structures might take place upon invasion of a prokaryote cell by foreign DNA.
+ The mechanism of CRISPR generation is not known, but the cas genes, which are invariably closely linked to the CRISPR structures, are presumably involved in this process  
+ The process should involve the formation of segments of a defined size, destined to become spacers, and their linkage to the repeated element. The presence of exonuclease motifs of the recB type in the cas4 genes, and the HNH endonuclease motif in the cas5 genes prompts us to suggest that the segments are formed by a nucleolytic activity, but in two different ways. The __Cas5__(Cas9) endonuclease might excise the segments by __internal DNA cleavage__, possibly directed by the short conserved sequence that we identified in the extrachromosomal donor elements at a constant position relative to the spacer-matching region.  
+ Here we report a correlation between the number of spacers in a locus and the resistance of S. thermophilus to phage infection, suggesting that CRISPRs can have a different biological role, protecting the bacteria against phage attack.  
+ A possible mechanism is via anti-sense RNA inhibition of phage gene expression
    +  spacers that are homologous to phage coding sequences can have either of the two orientations within a CRISPR locus, and thus give rise to anti-sense RNA, irrespective of the direction of locus transcription.  
    + CRISPR loci do appear to be transcribed and can thus generate the anti-sense RNA  
    + anti-sense RNA inhibits phage propagation  
+ A truncated repeat(PAM) in the orientation opposite to that found within the CRISPR structure is often present upstream of the cluster.

---

_2005_ [__CRISPR elements in Yersinia pestis acquire new repeats by preferential uptake of bacteriophage DNA, and provide additional tools for evolutionary studies__](http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.27437-0#tab2)  


_INTRO_   

+ The repeated sequence possesses characteristic features suggesting that it could be a target for DNA-binding proteins  
+ The existence of several conserved genes (__Cas__) in the vicinity of CRISPRs, potentially involved in DNA recombination and repair, is an additional argument for a physiological role.  
+ CRISPRs are thought to increase in size by duplicating the constant sequence and adding at least one new spacer by a still unknown mechanism.  
+ In __Yersinia pestis__, Jansen reported the existence of __three__ CRISPR elements with the same repeat sequence.

---

_2007_ [__CRISPR provides acquired resistance against viruses in prokaryotes__](http://www.ncbi.nlm.nih.gov/pubmed/17379808)  

_ABSTRACT_  

+ After viral challenge, bacteria integrated new spacers derived from phage genomic sequences.   
+ Removal or addition of particular spacers modified the phage-resistance phenotype of the cell.   
+ Thus, CRISPR, together with associated cas genes, provided resistance against phages, and resistance specificity is determined by spacer-phage sequence similarity.   


_INTRO_  

+ Streptococcus thermophilus is a low G+C Gram-positive bacterium and a key species exploited in the formulation of dairy culture systems for the production of yogurt and cheese.
+ We analyzed the CRISPR sequences of various S. thermophilus strains, including closely related industrial strains and phage-resistant variants  
+ phage sensitivity appeared to be correlated with CRISPR1 spacer content. Specifically, spacer content was nearly identical between parental strains and phage-resistant derivatives, except for additional spacers present in the latter.

_RESULTS_

_CRISPR loci are altered during the natural generation of phage-resistant mutants_  

A phage-host model system was selected, consisting of a phage-sensitive __wild-type__ S. thermophilus strain widely used in the dairy industry. Nine __phage-resistant mutants__ were generated independently by challenging the WT strain with phage 858, phage 2972, or simultaneously with both and their CRISPR loci analyzed. Differences were consistently observed at the CRISPR1 locus, where 1 to 4 additional spacers were inserted next to the 32 spacers present in the WT strain. The addition of new spacers in response to phage infection seemed to be polarized toward one end of the CRISPR1 locus.  

![S. thermophilus + phage CRISPR locus overview](https://d2ufo47lrtsv5s.cloudfront.net/content/sci/315/5819/1709/F1.large.jpg?width=800&height=600&carousel=1)

Streptococcus thermophilus CRISPR1 locus overview, newly acquired spacers in phage-resistant mutants, and corresponding phage sensitivity. The CRISPR1 locus of __DGCC7710__ (WT) is at the top. The repeat-spacer region of WT is in the middle: repeats (black diamonds), spacers (numbered gray boxes), leader (L, white box), and terminal repeat (T, black diamond).

Sequence analysis of the additional spacers inserted in the CRISPR1 locus of the various phage-resistant mutants revealed similarity to sequences found within the genomes of the phages used in the challenge. Interestingly, similarities were observed throughout the phage genomes, in most functional modules, both on the coding and noncoding strands. No particular sequence, gene, or functional group seemed to be targeted specifically.

![newly acquired spacer sequence homology to viruses](https://d2ufo47lrtsv5s.cloudfront.net/content/sci/315/5819/1709/F2.large.jpg?width=800&height=600&carousel=1)

S. thermophilus phage genome maps with the position of sequences similar to the acquired CRISPR1 spacers of the phage-resistant mutants.

> These results reveal that, on becoming resistant to bacteriophages, the CRISPR1 locus was modified by the integration of novel spacers, apparently derived from phage DNA. These findings suggest that the presence of a CRISPR spacer identical to a phage sequence provides resistance against phages containing this particular sequence.

_CRISPR spacer defines phage resistance_

We altered the CRISPR1 locus by adding and deleting spacers and tested subsequent strain sensitivity to phages.  
+ We __removed__ the spacers and repeats in the CRISPR1 locus of strain `WTΦ858+S1S2` and replaced them with a single repeat without any spacer. The resulting strain `WTΦ858(+S1S2)ΔCRISPR1` was sensitive to phage 858, which indicated that the phage resistance of the original phage-resistant mutant (`WTΦ858+S1S2`) was probably linked to the presence of `S1` and `S2`
+ To address the critical question of whether __adding spacers__ provides novel phage resistance, we replaced the CRISPR1 locus of strain `WTΦ2972+S4` with a version containing only spacers `S1` and `S2`. Remarkably, the resulting strain `WTΦ2972+S4::pS1S2` gained resistance to phage 858, which suggested that these two spacers have the ability to provide phage resistance de novo

![removal of new spacer alters phage sensitivity](https://d2ufo47lrtsv5s.cloudfront.net/content/sci/315/5819/1709/F3.large.jpg?width=800&height=600&carousel=1)

_CONCLUSIONS_  

Overall, prokaryotes appear to have evolved a nucleic acid–based “immunity” system whereby specificity is dictated by the CRISPR spacer content, while the resistance is provided by the Cas enzymatic machinery. 7

---

_2008_ [__Small CRISPR RNAs guide antiviral defense in prokaryotes__](http://www-ncbi-nlm-nih-gov.myaccess.library.utoronto.ca/pubmed/18703739)  

_ABSTRACT_  
Prokaryotes acquire virus resistance by integrating short fragments of viral nucleic acid into clusters of regularly interspaced short palindromic repeats (CRISPRs). Here we show how virus-derived sequences contained in CRISPRs are used by CRISPR-associated (Cas) proteins from the host to mediate an __antiviral response__ that counteracts infection. After transcription of the CRISPR, a complex of Cas proteins termed __Cascade__ cleaves a __CRISPR RNA precursor__ in each repeat and retains the cleavage products containing the virus-derived sequence. Assisted by the helicase Cas3, these mature CRISPR RNAs then serve as __small guide RNAs__ that enable Cascade to interfere with virus proliferation. Our results demonstrate that the formation of mature guide RNAs by the CRISPR RNA endonuclease subunit of Cascade is a mechanistic requirement for antiviral defense.  


_RESULTS_    

_Composition of Cascade Complex_  

The Escherichia coli K12 CRISPR/cas system comprises eight cas genes: `cas3` (predicted HD-nuclease fused to a DEAD-box helicase), five genes designated `casABCDE`, `cas1` (predicted integrase), and the endoribonuclease gene `cas2`.  each Cas protein was tagged at both the N and C terminus. Affinity purification of the tagged component enabled the identification of a protein complex composed of five Cas proteins: CasA, CasB, CasC, CasD, and CasE. The complex, denoted __Cascade__ (CRISPR-associated complex for antiviral defense), could be isolated from E. coli lysates.

![Cascade complex](https://d2ufo47lrtsv5s.cloudfront.net/content/sci/321/5891/960/F1.large.jpg?width=800&height=600&carousel=1)  

_Cascade cleaves CRISPR RNA precursors into small RNAs of ∼57 nucleotides_  

The function of Cascade was studied by analyzing the effect of in-frame cas gene knockouts on the formation of transcripts of the CRISPR region in E. coli K12. Northern analysis of total RNA with single-stranded spacer sequences as a probe showed transcription of the CRISPR region in the direction downstream of the cas2 gene. Analysis of control strains (wild type and a non-cas gene knockout) revealed a small CRISPR-RNA (crRNA) product of ∼57 nucleotides. The small crRNAs seem to be cleaved from a multi-unit crRNA precursor (pre-crRNA) as is evident from the presence of two and three repeat-spacer units (∼120 and ∼180 nucleotides) that show up in the ΔcasA, ΔcasB, and ΔcasC strains. The __ΔcasE__ strain contained a large pre-crRNA, suggesting that the disruption of this gene prevents pre-crRNA cleavage.


![crisper cleaves rna precursor into RNA of 57nt](https://d2ufo47lrtsv5s.cloudfront.net/content/sci/321/5891/960/F2.large.jpg?width=800&height=600&carousel=1)  

... more results tl;tl

_CONCLUSIONS_  

Our results demonstrate that a complex of five Cas proteins is responsible for the maturation of pre-crRNA to small crRNAs that are critical for mediating an antiviral response. These mature crRNAs contain the antiviral spacer unit flanked by short RNA sequences derived from the repeat on either side termed the 5′ and 3′ handle, which may serve as conserved binding sites for Cascade subunits. The Cascade-bound crRNA serves as a guide to direct the complex to viral nucleic acids to mediate an antiviral response. We hypothesize that crRNAs target virus DNA, because anti-λ CRISPRs of both polarities lead to a reduction of sensitivity to the phage.   

> We conclude that the transcription of CRISPR regions—and the cleavage of pre-crRNA to mature crRNAs by Cas proteins—is the molecular basis of the antiviral defense stage of the CRISPR/cas system, which enables prokaryotes to effectively prevent phage predation.

---

_2010_ [__The CRISPR/Cas bacterial immune system cleaves bacteriophage and plasmid DNA__](http://www.nature.com/nature/journal/v468/n7320/full/nature09523.html)  

_ABSTRACT_  

+ Here we show that the Streptococcus thermophilus CRISPR1/Cas system can also naturally acquire spacers from a self-replicating plasmid containing an antibiotic-resistance gene, leading to plasmid loss. Acquired spacers that match antibiotic-resistance genes provide a novel means to naturally select bacteria that cannot uptake and disseminate such genes.
+ We also provide in vivo evidence that the CRISPR1/Cas system specifically cleaves plasmid and bacteriophage double-stranded DNA within the proto-spacer, at specific sites.


_INTRO_  

+ CRISPR/Cas system act in two steps  
    +  the __adaptation__ stage, where new spacers derived from foreign DNA (proto-spacers) are generally acquired at the leader end of the CRISPR locus   
    +  the __interference__ stage, where the CRISPR/Cas system targets either invading DNA10 or RNA13    

_RESULTS_ (uncomplete)  

_CRISPR/Cas system cuts viral DNA in the proto-spacers_  

To test whether the CRISPR/Cas system cleaves other invading DNA within the proto-spacer, we analysed the fate of bacteriophage DNA during the infection of __bacteriophage-insensitive mutants__ containing bacteriophage-derived spacers. We used the virulent streptococcal `bacteriophage 2972`, its host `DGCC7710`, and three previously described bacteriophage-insensitive mutants derived from DGCC7710 following challenge with virulent `bacteriophages 2972` and/or `858`, namely `DGCC7710Φ2972+S4`, `DGCC7710Φ2972+S7` and `DGCC7710Φ2972+S4Φ858+S32`. These bacterial strains were infected with `bacteriophage 2972` and total DNA was extracted from bacteriophage-infected cells sampled over time.


__Southern hybridizations__ of the DNA throughout the time course of infection of the bacteriophage-insensitive mutants with probes targeting regions close to the proto-spacers revealed that the bacteriophage DNA is rapidly cleaved by the CRISPR/Cas machinery. The cleavage site in the bacteriophage genome seemed to be within or in the vicinity of the proto-spacer during the infection of bacteriophage-insensitive mutants, whereas no cleavage of bacteriophage DNA was observed in the sensitive strain DGCC7710. The __inactivation of cas5__ restored bacteriophage sensitivity (efficiency of plaquing of 1) and no bacteriophage DNA cleavage occurred, confirming the involvement of cas5 in interference. In contrast, the __inactivation of cas7__ did not affect bacteriophage resistance (efficiency of plaquing of 10−6) and bacteriophage DNA was cleaved

---

_2011_ [__CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III__](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3070239/)

---


_2013_ [__CRISPR-Cas9 harnessed for genome editing__](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3795411/)  

_ABSTRACT_
The type II prokaryotic CRISPR (clustered regularly interspaced short palindromic repeats)/Cas adaptive immune system has been shown to facilitate RNA-guided site-specific DNA cleavage. We engineered two different type II CRISPR/Cas systems and demonstrate that Cas9 nucleases can be directed by short RNAs to induce precise cleavage at endogenous genomic loci in human and mouse cells. Cas9 can also be converted into a nicking enzyme to facilitate homology-directed repair with minimal mutagenic activity. Lastly, multiple guide sequences can be encoded into a single CRISPR array to enable simultaneous editing of several sites within the mammalian genome, demonstrating easy programmability and wide applicability of the RNA-guided nuclease technology.
